src/styles/_forms.scss

```diff
@@ -43,6 +44,16 @@
     margin: 6px 0 0;
 }
 
+.form__group--helper {
```

Comment:
Shouldn't we use this class in the component instead of adding specific css properties there?

Reply:
this is for inputs in specific, and also needed to set the `dot-field-helper` position. better here to keep styles related to  helper in a single place ?